src/database/queries/sessions.js

```diff
@@ -593,10 +593,11 @@ exports.getUpcomingSessions = async (page, limit, search, userId) => {
 	}
 }
 
-exports.findAndCountAll = async (filter, options = {}) => {
+exports.findAndCountAll = async (filter, attributes, options = {}) => {
```

Comment:
@sumanvpacewisdom set default attributes  ={}, so that it won't break, if we didn't pass anything

Reply:
@sumanvpacewisdom add attributes as last argument otherwise it will break existing reference